packages/venia-concept/src/util/__mocks__/getRandomColor.js

```diff
@@ -0,0 +1,7 @@
+const getRandomColor = () =>
+    Array.from({ length: 3 }, () => Math.floor(Math.random() * 255)).join(',');
+
+// The primary reason for the existence of this mock is to return a non-random
+// memoized color.
+export const memoizedGetRandomColor = () => '123,123,123';
+export default getRandomColor;
```

Comment:
Is there any reason `getRandomColor` needs to have an actual implementation? Can it just do the same thing as `memoizedGetRandomColor` ?

A mock containing randomly generated anything is a red flag, and its less code we'll have to maintain in the end.

Reply:
> Is there any reason `getRandomColor` needs to have an actual implementation?

Are you asking about this mock file or in the more general sense?

> A mock containing randomly generated anything is a red flag

Definitely, I just wanted to match the actual implementation's export signature. I could just do `export default memoizedGetRandomColor as getRandomColor;`. Whatcha think?

Reply:
Yep, love it 👍